### ISSUE TITLE
Two errors when running PSPKI on basic CA

### ISSUE THREAD

**Issue by PrzemyslawKlys** [PKISolutions/PSPKI#194]
When running 

```powershell
$CAHealthStatus = Get-EnterprisePKIHealthStatus -CertificateAuthority $CA
```

![image](https://github.com/PKISolutions/PSPKI/assets/15063294/c4f11136-2581-433a-874c-64b186fed0ea)

When running

```powershell
$CAEnrollmentPolicyServer = Get-EnrollmentPolicyServerClient
```

![image](https://github.com/PKISolutions/PSPKI/assets/15063294/ff47f296-27fa-4ce3-85aa-46f79d04f7ec)

**Comment by Crypt32** [comment]
I can confirm both issues.

**Comment by Crypt32** [comment]
fixed in v4.2
